services/ocr/main.py

```diff
@@ -20,12 +20,19 @@ def callback(ch, method, properties, body):
         nparr = np.frombuffer(base64.b64decode(data["img"]), np.uint8)
         img = cv2.imdecode(nparr, cv2.IMREAD_COLOR)
 
-        out = Engine.process(img, int(data["week_number"]), connection2, data["id"])
-
-        channel.queue_declare(queue=f"scan:shift")
-        channel.basic_publish(exchange='',
-                            routing_key=f"scan:shift",
-                            body=json.dumps({"status": "PROCCESED", "scan_id": data["id"]}))
+        try:
+            out = Engine.process(img, int(data["week_number"]), connection2, data["id"])
+
+            channel.queue_declare(queue=f"scan:shift")
+            channel.basic_publish(exchange='',
+                                routing_key=f"scan:shift",
+                                body=json.dumps({"status": "PROCCESED", "scan_id": data["id"]}))
+            
+        except Exception as e:
+            channel.queue_declare(queue=f"scan:shift")
+            channel.basic_publish(exchange='',
+                                routing_key=f"scan:shift",
+                                body=json.dumps({"status": "Error", "errors": e, "scan_id": data["id"]}))
```

Comment:
Je tohle list errorů nebo jenom jeden error? Já to totiž posílám jako list, tak by bylo asi dobré být konzistentní. A status posílám jako "ERROR"

Reply:
Jen jeden error, chceš to teda:
`{"status": "Error", "errors": [Tady list těch errorů], "scan_id": "IDčko"}`
?

Reply:
No já to mám například takhle
```typescript
this.scanQueue.publishMessage({
  status: "ERROR",
  errors: ["postgres/no-absences-found"],
  scan_id,
});
```

Takže mít ten error capslockem a errors nějakej string kterej půjde nějak hezky na frontendu přeložit